### PR TITLE
Add cross condition in game over conditions

### PR DESCRIPTION
**Pull request by tsu-nera**
add cross condition in game over conditions. #1

### CODE REVIEW

File: gym_tic_tac_toe/envs/tic_tac_toe_env.py [hunk 1]
```diff
@@ -33,9 +33,11 @@ def _step(self, action):
 
         # check game over
         for i in range(3):
-            # horizontals and verticals
-            if ((board[i * 3] == p and board[i * 3 + 1] == p and board[i * 3 + 2 ] == p)
-                or (board[i + 0] == p and board[i + 3] == p and board[i + 6] == p)):
+            # horizontals and verticals and cross
```
Comment (nczempin) [inline gym_tic_tac_toe/envs/tic_tac_toe_env.py]: "diagonals" instead of "cross"

File: gym_tic_tac_toe/envs/tic_tac_toe_env.py [hunk 2]
```diff
@@ -33,9 +33,11 @@ def _step(self, action):
...
+            if ((board[i * 3] == p and board[i * 3 + 1] == p and board[i * 3 + 2] == p)
+                or (board[i + 0] == p and board[i + 3] == p and board[i + 6] == p)
+                or (board[0] == p and board[4] == p and board[8] == p)
```
Comment (nczempin) [inline gym_tic_tac_toe/envs/tic_tac_toe_env.py]: checking the diagonals should be outside the i loop

**Review by nczempin — CHANGES_REQUESTED** [review]

File: gym_tic_tac_toe/envs/tic_tac_toe_env.py (hunk 2 above)
Comment (tsu-nera) [reply gym_tic_tac_toe/envs/tic_tac_toe_env.py]: You are right. My commit is not efficient.

**Review by tsu-nera — COMMENTED** [review]
Thank you for reviewing. I modified as requested.
1. diagonals is better than cross.
2. check condition outside of the loop is more efficient.

**Review by nczempin — APPROVED** [review]

**Pull request merged**